v2/thirdparty/common-customizations/saml/with-boxyhq/integration-steps.mdx

```diff
@@ -233,6 +233,21 @@ curl --location --request PUT '^{coreInjector_uri_without_quotes}/recipe/multite
 
 ## 6) Configure the SAML provider for the tenant
 
+To configure SAML login with SuperTokens, ensure that you use the correct provider name in the third-party config.
```

Comment:
Move this section after the code snippet. And then mention that replace the `<provider-name>` in the code snippet above with one of the following